backends/tfhe-cuda-backend/src/cuda_bind.rs

```diff
@@ -311,6 +311,40 @@ extern "C" {
         num_samples: u32,
     );
 
+    /// This scratch function allocates the necessary amount of data on the GPU for
+    /// the public function packing keyswitch implementation on 64-bit
```

Comment:
the naming might have been discussed already but I don't recall it being the "public functional" packing KS 🤔 in core I don't recall us having a function anywhere, are you implmenting the application of a function in the GPU ?

Reply:
In the core implementation we don't use the "public functional" preffix, but that's the official name for the packing keyswitch we have there. I will adjust GPU code to follow the same convention.

Reply:
Huh, okay I’m just surprised as there is no explicit function 😅